subtests/docker_cli/kill_stress/kill_utils.py

```diff
@@ -113,9 +113,9 @@ def _init_container_attached(self, name):
         else:
             subargs = []
         subargs.append(name)
-        container = AsyncDockerCmd(self, 'attach', subargs)
-        self.sub_stuff['container_cmd'] = container  # overwrites finished cmd
-        container.execute()
```

Comment:
R0204:116,8: kill_base._init_container_attached: Redefinition of container type from dockertest.dockercmd.DockerCmd to dockertest.dockercmd.AsyncDockerCmd